sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventHubsEventSource.cs

```diff
@@ -1093,6 +1093,308 @@ public virtual void ReadAllEventsComplete(string eventHubName)
             }
         }
 
+        /// <summary>
+        ///   Indicates that an AMQP object was closed by the <c>FaultTolerantAmqpObject&lt;T&gt;</c> instance that manages it.
+        /// </summary>
+        ///
+        /// <param name="objectTypeName">The name of the AMQP object type being managed; normally this will be a type of AMQP link.</param>
+        /// <param name="identifier">The identifier of the client associated with the AMQP object.</param>
+        /// <param name="eventHubName">The name of the Event Hub that the AMQP object is associated with.</param>
+        /// <param name="consumerGroup">The name of the consumer group that the AMQP object is associated with.</param>
+        /// <param name="partitionId">The identifier of the partition that the AMQP object is associated with.</param>
+        /// <param name="errorMessage">The message for any terminal exception that may have occurred.</param>
+        ///
+        [Event(55, Level = EventLevel.Verbose, Message = "An AMQP object of type '{0}' was closed by the fault tolerant manager for client '{1}', Event Hub: {2}, Consumer Group: {3}, and partition: {4}.  Terminal error message: '{5}'")]
+        public virtual void FaultTolerantAmqpObjectClose(string objectTypeName,
+                                                         string identifier,
+                                                         string eventHubName,
+                                                         string consumerGroup,
+                                                         string partitionId,
+                                                         string errorMessage)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(55, objectTypeName ?? string.Empty, identifier ?? string.Empty, eventHubName ?? string.Empty, consumerGroup ?? string.Empty, partitionId ?? string.Empty, errorMessage ?? string.Empty);
+            }
+        }
+
+        /// <summary>
+        ///   Indicates that creating an AMQP connection has started.
+        /// </summary>
+        ///
+        /// <param name="endpoint">The service endpoint that the connection is being opened for.</param>
+        /// <param name="transportType">The type of transport being used for the connection</param>
+        ///
+        [Event(56, Level = EventLevel.Verbose, Message = "Beginning creation of an AMQP connection for '{0}' using the transport '{1}`.")]
```

Comment:
nit for consistency:
```suggestion
        [Event(56, Level = EventLevel.Verbose, Message = "Beginning creation of an AMQP connection for endpoint: '{0}' using the transport: '{1}`.")]
```